### PR TITLE
Refactor rgen and help

### PR DESCRIPTION
**Pull request by duongphammmm**
image

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/generate/GenerateResumeCommand.java [hunk 1]
```diff
@@ -77,65 +78,105 @@ public CommandResult execute(Model model) throws CommandException {
...
+     * @param skillsToAdd the list of {@code Skill} ids to be added to the .pdf file.
+     * @throws IOException
+     */
+    private void addSkills(Model model, List<Integer> skillsToAdd) throws IOException {
+        if (!skillsToAdd.isEmpty()) {
+            builder.addSectionTitle("SKILLS");
+            List<Skill> skills = skillsToAdd.stream()
+                    .map(x -> model.hasSkillId(x) ? model.getSkillById(x) : null)
+                    .filter(Objects::nonNull)
+                    .collect(Collectors.toList());
```
Comment (wardetu) [inline src/main/java/seedu/address/logic/commands/generate/GenerateResumeCommand.java]: Why are these two methods implemented differently ?
They are supposedly congruent, aren't they?

**Review by wardetu — CHANGES_REQUESTED** [review]
Looks solid, would need to see it in action. I think you can upload a screenshot in the description box.

`PdfBuilder.java` contains a lot, and I mean A LOT, of public methods. They should be set to private if you don't intend other classes to use them.

File: src/main/java/seedu/address/logic/commands/generate/GenerateResumeCommand.java (hunk 1 above)
Comment (duongphammmm) [reply src/main/java/seedu/address/logic/commands/generate/GenerateResumeCommand.java]: for skills, i want to print skills in decreasing order of their level 😄

**Review by wardetu — APPROVED** [review]
LGTM

**Review by helloImHai — APPROVED** [review]
All fine and dandy 💃💃

**Pull request merged**